src/validator/MatchValidator.cpp

```diff
@@ -207,6 +194,66 @@ Status MatchValidator::validateReturn(MatchReturn *ret) {
     }
     NG_RETURN_IF_ERROR(validateAliases(exprs));
 
+    auto *skipExpr = ret->skip();
+    auto *limitExpr = ret->limit();
+    int64_t skip = 0;
+    int64_t limit = std::numeric_limits<int64_t>::max();
+    if (skipExpr != nullptr) {
+        if (skipExpr->kind() != Expression::Kind::kConstant) {
```

Comment:
it can't be a mathematical expression？maybe you can use `evaluableExpr` to check it.

Reply:
Maybe next time.

Reply:
done.